packages/fast-components-react-msft/src/metatext/metatext.spec.tsx

```diff
@@ -26,6 +26,14 @@ describe("metatext", (): void => {
         expect((MSFTMetatext as any).name).toBe(MSFTMetatext.displayName);
     });
 
+    test("should not throw if managedClasses are not provided", () => {
+        expect(
+            () => {
+                shallow(<Metatext />);
+            }
+        ).not.toThrow();
+    });
```

Comment:
This is a MSFT instance - are we sure that managed classes are NOT being provided here? We don't currently need to provide them to MSFT instances due to the work of the JSS manager.

Reply:
oops, no this should be an unmanaged instance